regipy/plugins/ntuser/winscp_saved_sessions.py

```diff
@@ -24,7 +24,7 @@ def _get_winscp_saved_sessions(self, subkey_path):
             return
 
         for winscp_saved_session in sessions_sk.iter_subkeys():
-            values = {underscore(x.name): x.value for x in
+            values = {x.name: x.value for x in
```

Comment:
You prefer not to normalize these fields? fine by me :)

Reply:
yes :) because in Dexter all the values are inserted to a nested field of values:{name , data} and it will be more accurate that way